docs/ci-cd-integration/deploy-from-github.md

```diff
@@ -102,7 +102,98 @@ When you commit files to your project's repo on the "main" branch, Github Action
 
 ### Alternative method (more efficient)
 
-Alternatively, you can even build the Docker image on Github and just deploy the built artifact to your CapRover instance. This will help as it does not consume RAM and CPU from your CapRover instance to build your image. See this [comment](https://github.com/caprover/caprover/issues/1514#issuecomment-1250369318) for more details.
+Alternatively, you can even build the Docker image on Github and just deploy the built artifact to your CapRover instance. This will help as it does not consume RAM and CPU from your CapRover instance to build your image.
+
+In order to achieve this we will need to take the following steps to build the Docker image using GitHub Actions, store it using GitHub Packages, and then deploy it to CapRover.
+
+#### Create a GitHub Personal Access Token
+You will need to create a GitHub Personal Access Token with write permission for packages.
+
+GitHub has a great guide on creating a personal access token if you have not before. Here is the link: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
+
+#### Create a New App
+If you do not have an app already on CapRover, create one using the instructions [here](### Create a new App)
```

Comment:
`### Create a new App` where do these links lead to? I don't see a linked page here. Same with the other ones below:
- Enable App Token
- Add the Github Secrets
- Add files to project

Reply:
They link to the section above in the page. In this specific example it links to the ### Create a new App section on line 11.